packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js

```diff
@@ -41,204 +51,109 @@ function setupRegularPresetTaskFactory(
   function setupRegularPresetTask() {
     return new Listr([
       {
-        task: (ctx) => {
-          ctx.config = configFile.getConfig(ctx.preset);
-        },
-      },
-      {
-        title: 'Set node type',
+        title: 'Node type',
         task: async (ctx, task) => {
-          if (ctx.nodeType === undefined) {
-            ctx.nodeType = await task.prompt([
-              {
-                type: 'select',
-                message: 'Select node type',
-                choices: NODE_TYPES,
-                initial: NODE_TYPE_MASTERNODE,
-              },
-            ]);
+          const nodeTypeName = await task.prompt([
+            {
+              type: 'select',
+              // Keep this order, because each item references the text in the previous item
+              header: `  The Dash network consists of several different node types:
+    Full node                   - Host the full Dash blockchain (no collateral)
+    Masternode                  - Full node features, plus Core services such as
+                                  ChainLocks and InstantSend (1000 DASH collateral)
+    High-performance full node  - Full node features, plus host a full copy of the
+                                  Platform blockchain (no collateral)
```

Comment:
Such nodes will only have the blockchain? Or will they be able to provide read-only platform services if someone wants to only request data from their own node instead of the masternodes?

Reply:
Yeah, you can interact with your local node(s) to increase your app's performance and security (you don't need light clients in this case).